telco_ref_design_specs/core/telco-core-rds-use-cases.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="telco-ran-rds-overview"]
+= Telco core use case model overview
+include::_attributes/common-attributes.adoc[]
+:context: ran-core-design-overview
+
+toc::[]
+
+The use model for a telco core cluster provides a platform supporting large scale telecom applications including control plane functions like signaling and aggregation, and some centralized data plane for example user plane functions (UPF) functions running on commodity hardware. For examples of these functions see the architecture diagram. These functions generally require scalability, complex networking support, resilient software-defined storage, and support performance requirements that are less stringent and constrained than far-edge deployments like RAN.
```

Comment:
```suggestion
The use model for a telco core cluster provides a platform supporting large-scale telco applications including control plane functions like signaling and aggregation, and some centralized data plane functions. For example, user plane functions (UPF) running on commodity hardware. These functions generally require scalability, complex networking support, resilient software-defined storage, and support performance requirements that are less stringent and constrained than far-edge deployments like RAN.
```

"For examples of these functions see the architecture diagram." There is no diagram?